src/config.js

```diff
@@ -223,5 +223,9 @@ export default {
       prefix: process.env.LINKCHECK_PREFIX || "linkcheck-1",
       ttl: process.env.LINKCHECK_TIME_TO_LIVE_SECONDS || 60 * 60,
     },
+    userdata: {
+      url: process.env.USERDATA_URL,
+      ttl: process.env.USERDATA_TIME_TO_LIVE_SECONDS || 60 * 60,
```

Comment:
jeg synes vi skal cache kortere - hvis bruger laver en ændring forventer de at se den inden for kort tid

Reply:
done

Reply:
Har sat det til 5 sekunder. Er det fint ? eller hvad tænker ud?:)